src/model/neuroevolution/neuroevolution.py

```diff
@@ -16,63 +16,32 @@ class Neuroevolution:
     Genetic algorithm.
     """
 
-    _INDIVIDUALS: int = 20
+    _iteration_counter = 1
+    _INDIVIDUALS: int = 70
     """
     Amount of individuals per generation, minimum 3.
     """
 
-    _GOLDEN_TICKETS: int = 4
-    """
-    Amount of individuals allowed to reproduce for sure.
-    """
-
-    _MAX_PARENTS: int = _INDIVIDUALS // 3
-    """
-    Maximal parents count.
-    """
-
-    @staticmethod
-    def _reproduction_probability(
-        individual: AdultIndividual, bound_adaptation: float
-    ) -> float:
-        """
-        Probability of reproduction for every given individual
-        without golden ticket according to bound adaptation.
-        """
-        return (individual.adaptation / bound_adaptation) ** 3
```

Comment:
Such approach provides the algorithm with some adaptation ability. I would not remove whis concept (we dont want to lose code generality), but just reimplement this method -- just return 0.0 each time.

Reply:
Done